<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoConfigStreamPage</class>
 <widget class="QWidget" name="AutoConfigStreamPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="widget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="topLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>87</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="serviceLabel">
        <property name="text">
         <string>Basic.AutoConfig.StreamPage.Service</string>
        </property>
        <property name="buddy">
         <cstring>service</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="serviceWidget">
        <layout class="QHBoxLayout" name="serviceWidgetLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="service"/>
         </item>
         <item>
          <widget class="UrlPushButton" name="moreInfoButton">
           <property name="text">
            <string>Basic.AutoConfig.StreamPage.MoreInfo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>12</width>
          <height>12</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="loginPage">
      <layout class="QFormLayout" name="loginPageLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>87</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="connectAccount">
           <property name="text">
            <string>Basic.AutoConfig.StreamPage.ConnectAccount</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>87</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="useStreamKey">
           <property name="text">
            <string>Basic.AutoConfig.StreamPage.UseStreamKey</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="streamkeyPage">
      <layout class="QFormLayout" name="streamkeyPageLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="serverLabel">
         <property name="text">
          <string>Basic.AutoConfig.StreamPage.Server</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QStackedWidget" name="serverStackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="servicePage">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="server"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="customPage">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="customServer"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="streamKeyLabel">
         <property name="text">
          <string>Basic.AutoConfig.StreamPage.StreamKey</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="buddy">
          <cstring>key</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="streamKeyWidget">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="key">
            <property name="inputMask">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="show">
            <property name="text">
             <string>Show</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="UrlPushButton" name="streamKeyButton">
            <property name="text">
             <string>Basic.AutoConfig.StreamPage.GetStreamKey</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="connectAccount2">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Basic.AutoConfig.StreamPage.ConnectAccount</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="connectedAccountLabel">
         <property name="text">
          <string>Basic.AutoConfig.StreamPage.ConnectedAccount</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="leftMargin">
          <number>7</number>
         </property>
         <property name="rightMargin">
          <number>7</number>
         </property>
         <item>
          <widget class="QLabel" name="connectedAccountText">
           <property name="font">
            <font>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Auth.LoadingChannel.Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="disconnectAccount">
           <property name="text">
            <string>Basic.AutoConfig.StreamPage.DisconnectAccount</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="useStreamKeyAdv">
         <property name="text">
          <string>Basic.AutoConfig.StreamPage.UseStreamKeyAdvanced</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="bitrateLabel">
         <property name="text">
          <string>Basic.Settings.Output.VideoBitrate</string>
         </property>
         <property name="buddy">
          <cstring>bitrate</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="bitrate">
         <property name="suffix">
          <string notr="true"/>
         </property>
         <property name="minimum">
          <number>500</number>
         </property>
         <property name="maximum">
          <number>51000</number>
         </property>
         <property name="value">
          <number>2500</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="preferHardware">
         <property name="toolTip">
          <string>Basic.AutoConfig.StreamPage.PreferHardwareEncoding.ToolTip</string>
         </property>
         <property name="text">
          <string>Basic.AutoConfig.StreamPage.PreferHardwareEncoding</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="doBandwidthTest">
         <property name="text">
          <string>Basic.AutoConfig.StreamPage.PerformBandwidthTest</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="useMultitrackVideo">
         <property name="text">
          <string>Basic.AutoConfig.StreamPage.UseMultitrackVideo</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="multitrackVideoInfo">
         <property name="text">
          <string>MultitrackVideo.Info</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <enum>QSizePolicy::MinimumExpanding, QSizePolicy::Minimum</enum>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>6</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="1">
        <widget class="QGroupBox" name="region">
         <property name="title">
          <string>BandwidthTest.Region</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="0">
           <widget class="QCheckBox" name="regionAsia">
            <property name="text">
             <string>BandwidthTest.Region.Asia</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="regionUS">
            <property name="text">
             <string>BandwidthTest.Region.US</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="regionEU">
            <property name="text">
             <string>BandwidthTest.Region.EU</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="regionOther">
            <property name="text">
             <string>BandwidthTest.Region.Other</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UrlPushButton</class>
   <extends>QPushButton</extends>
   <header>components/UrlPushButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>connectAccount2</sender>
   <signal>clicked()</signal>
   <receiver>connectAccount</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
